e2etest/zt_preserve_access_tier_test.go

```diff
@@ -26,6 +26,19 @@ import (
 	"testing"
 )
 
+//func TestTier_UploadCold(t *testing.T) {
+//	RunScenarios(t, eOperation.Copy(), eTestFromTo.Other(common.EFromTo.LocalBlob()), eValidate.Auto(), anonymousAuthOnly, anonymousAuthOnly, params{
```

Comment:
if this is dead code, we can remove it.

Reply:
This is not dead code. This is a test that can't currently be ran and succeed, because this isn't supported by the service (yet).